js/glowingbear.js

```diff
@@ -233,7 +240,7 @@ weechat.controller('WeechatCtrl', ['$rootScope', '$scope', '$store', '$timeout',
         $scope.notifications = notifications.unreadCount('notification');
         $scope.unread = notifications.unreadCount('unread');
 
-        if (settings.useFavico && $rootScope.favico) {
+        if (window.cordova === undefined && settings.useFavico && $rootScope.favico) {
```

Comment:
`settings.useFavico` should never be true in cordova so I don't think this is needed